
This message causes numerous useless "bug reports", so silence it by default.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- linux/net/netfilter/xt_addrtype.c.orig	2017-11-12 20:46:13.000000000 +0200
+++ linux/net/netfilter/xt_addrtype.c	2018-02-06 20:26:49.854596642 +0200
@@ -203,7 +203,7 @@ static int addrtype_mt_checkentry_v1(con
 			return -EINVAL;
 		}
 		if ((info->source | info->dest) & XT_ADDRTYPE_BROADCAST) {
-			pr_err("ipv6 does not support BROADCAST matching\n");
+			/* pr_err("ipv6 does not support BROADCAST matching\n"); */
 			return -EINVAL;
 		}
 	}
